_gsocproposals/2018/proposal_HAhRD3D-Clustering.md

```diff
@@ -3,7 +3,9 @@ title: 3D-Clustering
 layout: gsoc_proposal
 project: HAhRD
 year: 2018
-organization: LLR
+organization:
+  - LLR
+  - CMS
```

Comment:
CMS is not an organization but a project. Please move it to the project section.